package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "opentok-network-test-js",
+  "name": "@vonage/video-client-network-test",
   "version": "3.1.5",
```

Comment:
I am tempted to leave this as 3.1.5, so that it is in line with the OpenTok version. Or does that seem odd?